tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,20 @@
+export function followTheSpy(arr) {
+  let result = '';
+  result = result.concat(arr[0][0] + ', ' + arr[0][1]);
```

Comment:
why not push to array which you will join after, so you will not have to deal with `,`